undefined statsd NoOpClient

Hi i'm experiencing this issue with v1.28
pkg/mod/gopkg.in/!data!dog/dd-trace-go.v1@v1.28.0/ddtrace/tracer/option.go:207:16: undefined: statsd.NoOpClient
pkg/mod/gopkg.in/!data!dog/dd-trace-go.v1@v1.28.0/profiler/options.go:100:19: undefined: statsd.NoOpClient

How to reproduce
```
go get gopkg.in/DataDog/dd-trace-go.v1/...
```

Env
go version go1.15.7 darwin/amd64
Apple M1

Do i miss something? i think this need to revisit and fix

--- a/go.mod
+++ b/go.mod
@@ -3,6 +3,7 @@ module gopkg.in/DataDog/dd-trace-go.v1
 go 1.12
 
 require (
+	github.com/DataDog/datadog-go v4.4.0+incompatible
 	github.com/google/pprof v0.0.0-20210125172800-10e9aeb4a998
 	github.com/tinylib/msgp v1.1.2
 )

